api/service/explorer/storage_test.go

```diff
@@ -93,7 +93,8 @@ func TestDumpCommittee(t *testing.T) {
 	shardID := uint32(0)
 	epoch := uint64(0)
 	ins := GetStorageInstance("1.1.1.1", "3333", true)
-	ins.DumpCommittee(shardID, epoch, committee)
+	err = ins.DumpCommittee(shardID, epoch, committee)
+	assert.Nil(t, err, "should be nil")
```

Comment:
Unexpected errors in UTs should be printed out in order to let testers know about what went wrong.